crates/teloxide/src/utils/render/helper.rs

```diff
@@ -0,0 +1,61 @@
+//! A helprt trait for rendering text/caption and entities back to html or
+//! markdown
+
+use teloxide_core::types::Message;
+
+use super::Render;
+
+/// The [`RenderMessageTextHelper`] trait provides methods to generate HTML and
+/// Markdown representations of the text and captions in a Telegram message.
+pub trait RenderMessageTextHelper {
```

Comment:
Let's add newlines after each function signature in this trait (except the last one).